sherpa/bin/streaming_pruned_transducer_statelessX/beam_search.py

```diff
@@ -0,0 +1,296 @@
+from typing import List
+
+import k2
+import torch
+from decode import Stream
```

Comment:
Now `beam_search.py` does the decoding stuff.

Shall we rename `decode.py` to `stream.py`?

Reply:
Make sense, will add it in the coming commits

Reply:
Done